components/SlateRenderer/SlateRenderer.tsx

```diff
@@ -1,38 +1,30 @@
-import type { ComponentRenderers } from '@prezly/content-renderer-react-js';
-import { Contact, Renderer } from '@prezly/content-renderer-react-js';
-import type { Node } from '@prezly/slate-types';
+import { Component, Renderer } from '@prezly/content-renderer-react-js';
+import type { Node } from '@prezly/story-content-format';
 import {
-    BULLETED_LIST_NODE_TYPE,
-    CONTACT_NODE_TYPE,
-    HEADING_1_NODE_TYPE,
-    HEADING_2_NODE_TYPE,
-    LIST_ITEM_NODE_TYPE,
-    LIST_ITEM_TEXT_NODE_TYPE,
-    NUMBERED_LIST_NODE_TYPE,
-    PARAGRAPH_NODE_TYPE,
-} from '@prezly/slate-types';
+    ContactNode,
+    HeadingNode,
+    ListItemNode,
+    ListItemTextNode,
+    ListNode,
+    ParagraphNode,
+} from '@prezly/story-content-format';
+
 import '@prezly/content-renderer-react-js/styles.css';
+import { ContactCard, Heading, List, ListItem, ListItemText, Paragraph } from './components';
 
 interface Props {
     nodes: Node | Node[];
 }
 
-const components: ComponentRenderers = {
-    [LIST_ITEM_TEXT_NODE_TYPE]: ({ children }) => <>{children}</>,
-    [LIST_ITEM_NODE_TYPE]: ({ children }) => <li>{children}</li>,
-    [BULLETED_LIST_NODE_TYPE]: ({ children }) => <ul>{children}</ul>,
-    [NUMBERED_LIST_NODE_TYPE]: ({ children }) => <ol>{children}</ol>,
-    // // [DOCUMENT_NODE_TYPE]: ({ children, node }) => (
-    //     <section data-version={node.version}>{children}</section>
-    // ),
-    [HEADING_1_NODE_TYPE]: ({ children }) => <h2>{children}</h2>,
-    [HEADING_2_NODE_TYPE]: ({ children }) => <h3>{children}</h3>,
-    // [LINK_NODE_TYPE]: ({ children, node }) => <a href={node.href}>{children}</a>,
-    [PARAGRAPH_NODE_TYPE]: ({ children }) => <p>{children}</p>,
-    // [QUOTE_NODE_TYPE]: ({ children }) => <blockquote>{children}</blockquote>,
-    [CONTACT_NODE_TYPE]: ({ node }) => <Contact node={node} className="not-prose" />,
-};
-
 export function SlateRenderer({ nodes }: Props) {
-    return <Renderer nodes={nodes} components={components} />;
+    return (
+        <Renderer nodes={nodes} defaultComponents>
```

Comment:
The updated renderer now supports Story Bookmarks, but there is no components set up for it. Could you add it in this PR?

<img width="815" alt="image" src="https://user-images.githubusercontent.com/1641218/207258651-9c3a4346-b3a3-405e-a1bb-929ec1cdb155.png">